pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -500,8 +504,30 @@ func setResourcesV2(cgroupConfig *libcontainerconfigs.Cgroup) error {
 	if err := propagateControllers(cgroupConfig.Path); err != nil {
 		return err
 	}
-	allowAll := true
-	cgroupConfig.Resources.AllowAllDevices = &allowAll
+	cgroupConfig.Resources.Devices = []*libcontainerconfigs.DeviceRule{
+		{
+			Type:        'a',
+			Permissions: "rwm",
+			Allow:       true,
+			Minor:       libcontainerconfigs.Wildcard,
+			Major:       libcontainerconfigs.Wildcard,
+		},
+	}
+	cgroupConfig.Resources.Memory = -1
+	cgroupConfig.Resources.CpuWeight = 100
+	cgroupConfig.Resources.PidsLimit = -1
+
+	cpuset, err := ioutil.ReadFile(filepath.Join(cmutil.CgroupRoot, "cpuset.cpus.effective"))
+	if err != nil {
+		return fmt.Errorf("failed to read controllers from %q : %v", cmutil.CgroupRoot, err)
+	}
+	cgroupConfig.Resources.CpusetCpus = string(cpuset)
+	cgroupConfig.Resources.HugetlbLimit = []*libcontainerconfigs.HugepageLimit{
```

Comment:
Is this something new in this PR?

Reply:
Thanks for noticing it.  I've needed it for an older version of runc/libcontainer, it seems it is not needed anymore with master.  I'll drop this part.